Disable custom styling for <select> and <option> on Tier3 as well

diff --git modules/libpref/init/all.js modules/libpref/init/all.js
index 322d009da739..06e2fd7f664a 100644
--- modules/libpref/init/all.js
+++ modules/libpref/init/all.js
@@ -1359,8 +1359,8 @@ pref("dom.forms.autocomplete.formautofill", false);
 // Enable search in <select> dropdowns (more than 40 options)
 pref("dom.forms.selectSearch", false);
 // Allow for webpages to provide custom styling for <select>
-// popups. Disabled on Linux due to bug 1338283.
-#ifdef XP_LINUX
+// popups. Disabled on GTK due to bug 1338283.
+#ifdef MOZ_WIDGET_GTK
 pref("dom.forms.select.customstyling", false);
 #else
 pref("dom.forms.select.customstyling", true);
